### DISCUSSION TITLE
GitHub Classroom Bug - "Link to student" button links wrong account

### DISCUSSION THREAD

**Discussion by plbrault** [community/community#64808] [GitHub Education]
### Hi everyone,

I am experiencing a bug in GitHub Classroom. In the `Unlinked GitHub accounts` tab of the page that allows to manage the classroom roster, if I click on the `Link to student` button beside any of the unlinked accounts, and then select a student, it always links the first account in the list to that student instead of the one associated to the button I clicked on.

**Comment by bbenjamin** [comment]
I also ran into this - poking around to see if Classroom itself can be contributed to so I can provide a fix.

**Comment by HarshDekate** [comment]
HELLO
